Application/EdFi.Ods.Tests/EdFi.Ods.Api/Filters/InstanceIdContextFilterTests.cs

```diff
@@ -161,9 +163,10 @@ protected override void Arrange()
             public void Should_Valid_Instance_Id()
             {
                 A.CallTo(() => _instanceIdContextProvider.SetInstanceId(A<string>._)).MustHaveHappened();
```

Comment:
The code you added is just calling your fake, you can just change the previous line to verify it specifically called SetInstanceId with the _instanceId value.
```suggestion
                A.CallTo(() => _instanceIdContextProvider.SetInstanceId(_instanceId)).MustHaveHappened();
```